db/seeds.rb

```diff
@@ -136,8 +195,9 @@
 
 1000.times do
   city, state = cities_and_states.sample
+  name = refugee_shelter_names.sample
   coordinates = coordinates_by_city_state["#{city}, #{state}"]
   random_lat = rand(lat_min_range..lat_max_range)
   random_lon = rand(long_min_range..long_max_range)
```

Comment:
It looks like the coordinates being used in the actual FactoryBot creation do not utilize the city coordinates. Would the following work?
```
random_lat = rand(lat_min_range..lat_max_range) + coordinates[0]
random_lon = rand(long_min_range..long_max_range) + coordinates[1]
```